chunk_utils.py

```
@@ -0,0 +1,209 @@
+                chunk_id = hashlib.md5(f"{source_url}:{section_heading}".encode()).hexdigest()
+                chunk = Chunk(
+                    id=chunk_id,
+                    content=section_content,
+                    metadata={
+                        "heading": section_heading,
+                        "domain": domain,
+                        "word_count": len(section_content.split()),
+                        "char_count": len(section_content)
+                    },
+                    source_url=source_url,
+                    created_at=datetime.now().isoformat(),
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)